.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# https://help.github.com/en/articles/about-code-owners
+# Default reviewers for everything
+* @mnecas @arachmani @barpavel
```

Comment:
Please fill-in codeowners file according to current responsibilities. For example you can take a look to VDSM: https://github.com/oVirt/vdsm/blob/master/CODEOWNERS
Each of current maintaners already have his own field of responsoibility